Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/IFileSystem.cs

```diff
@@ -182,18 +105,12 @@ public ResultCode OpenFile(ServiceCtx context)
 
             string name = ReadUtf8String(context);
 
-            try
-            {
-                LibHac.Fs.IFile file = _fileSystem.OpenFile(name, mode);
+            Result rc = _fileSystem.OpenFile(out LibHac.Fs.IFile file, name, mode);
```

Comment:
```suggestion
            Result resultCode = _fileSystem.OpenFile(out LibHac.Fs.IFile file, name, mode);
```

and add an empty line after.